assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -11690,9 +11688,9 @@ private void initializeAssignment(SessionState state)
 			try {
 				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
 				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
-					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{contentreviewSiteMin, reviewServiceName}));
+					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
```

Comment:
No, I'm saying that the check was already in the code before this patch. This is what appears on that file before the new lines:

if(allowReviewService && allowLTIReviewService){
	Site st = null;
	try{

The new bit of code wasn't exactly modified for this issue, the change on that line is just related to the ordering of the properties in the message. So it's all ok for merging imo.